modules/end-user-guide/examples/snip_che-table-of-supported-editors.adoc

```diff
@@ -22,6 +14,14 @@
 | `che-incubator/che-pycharm/latest`
 | Community Edition - stable version
 
-| link:https://www.jetbrains.com/help/pycharm/quick-start-guide.html[PyCharm]
+| link:https://www.jetbrains.com/help/pycharm/quick-start-guide.html[JetBrains PyCharm Community Edition]
 | `che-incubator/che-pycharm/next`
-| Community Edition - development version
+| Development version
+
+| link:https://github.com/eclipse-che/che-theia[Che-Theia]
```

Comment:
Nick wants us to use "Eclipse Theia" according to https://issues.redhat.com/browse/CRW-3098, so I'm updating it here:
```suggestion
| link:https://github.com/eclipse-che/che-theia[Eclipse Theia]
```